larryTheGeek/RideApi
# Delivers [#158795315  #158795461] User should be able to sign in and out

What does this PR do?
Adds signup endpoint to the API

Description of Task to be completed
Adding a signup and login endpoint to allow users to sign up

POST /api/v1/signup
POST/api/v1/login

Tasks achieved in this PR

POST /api/v1/signup
POST/api/v1/login
How should one test this endpoint?

Clone this repo using git clone 
Check the ReadMe for instructions on setting up the project.
git checkout f1
Run pytest ridemyway/tests/test_v2/test_auth/test_signup.py to see the signup tests pass. Alternatively, 


**Background information**
The build fails because its a testing endpoints which are yet to be implemented. 

Resources & Libs
This Pivotal Tracker board was used to plan and manage the tasks.

Relevant PIvotal Tracker Stories?
#158795315 #158795461
